config/core_pools.json

```diff
@@ -1,6 +1,7 @@
 {
   "mainnet": {
     "0x05ff47afada98a98982113758878f9a8b9fdda0a000000000000000000000645": "weETH/rETH",
+    "0x1cce5169bde03f3d5ad0206f6bd057953539dae600020000000000000000062b": "ECLP-GYD-sDAI",
```

Comment:
This one may have been missed @mikebmikeb @Zen-Maxi should this be a core pool?

Reply:
@Zen-Maxi confirmed that it should be